src/components/IconMessage/__stories__/index.stories.mdx

```diff
@@ -1,85 +1,65 @@
-import { Canvas, Description, Meta, Story } from '@storybook/addon-docs/blocks'
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs/blocks'
 import IconMessage from '..'
 
-<Meta
-  title="Components/IconMessage"
-  component={IconMessage}
-  argTypes={{
-    icon: {
-      control: {
-        type: 'string',
-      },
-    },
-    variant: {
-      control: {
-        options: ['warning', 'info', 'success'],
-        type: 'select',
-      },
-    },
-  }}
-/>
+<Meta title="Components/IconMessage" component={IconMessage} />
 
 # IconMessage
 
+<Description>To merge in Alert</Description>
+
+## Variants
+
 <Description>
-  Set color using `variant` property with the same keywords of _variant_ colors
-  in `src/theming/colors.js` file. Set icon using `icon` property with the same
-  keywords of @scaleway/ui `Icon` component. Set texts using an array in `texts`
-  property. Each entry will be placed in a `Typography as=&quot;p&quot;`
-  component.
+  You can set the icon and text color by using `variant` prop
 </Description>
 
-## `variant`
-
 <Canvas>
-  <Story
-    name="warning"
-    args={{
-      icon: 'alert',
-      variant: 'warning',
-    }}
-  >
-    {args => (
-      <IconMessage {...args}>
-        LOREMINEM IPSUM
-        <br />
-        The way she moves, she&apos;s like a belly-dancer. She&apos;s shaking
-        that ass to the new Nelly jams. I think someone&apos;s at the door but I
-        don&apos;t think I&apos;mma answer.
-      </IconMessage>
-    )}
+  <Story name="Variant">
+    <IconMessage variant="warning">
+      LOREMINEM IPSUM
+      <br />
+      The way she moves, she&apos;s like a belly-dancer. She&apos;s shaking that
```

Comment:
We might have to update theses examples at some point 😂

Reply:
Yeah 😆